docs/cmd/prompt.md

```diff
@@ -1,57 +1,56 @@
 ## Description
-The `[prompt]` command starts an interactive gnmic prompt in which the same gNMI commands (`capabilities`, `get`, `set`, `subscribe`) can be executed with some additional auto completion features:
+The `prompt` command starts `gnmic` in an interactive prompt mode with the following auto-completion features:
 
-* Auto-completion for all commands names and their flags.
-* Auto-completion for flags with xpath values ( `--path`, `--prefix`, `--model`,...). The xpaths are generated from models suplied using `prompt` command `--file` and `--dir` (optional) flags.
-* Auto-completion for flags with enum values `--format`, `--encoding`, ...
-* Auto-completion for flags taking file or directory values.
+* All `gnmic` commands names and their flags are suggested.
+* Values for the flags that rely on YANG-defined data (like `--path`, `--prefix`, `--model`,...) will be dynamically suggested.  
+The auto-completions are generated from the YANG modules d with the `--file` and `--dir` flags.
+* Flags with the fixed set of values (`--format`, `--encoding`, ...) will get their values suggested.
+* Flags that require a file path value will auto-suggest the available files as the user types.
 
 
 ### Usage
 
 `gnmic [global-flags] prompt [local-flags]`
 
-
-### Flags 
+### Flags
 
 #### file
-
-File or directories pointing to yang modules from which to generate gnmi paths.
+A path to a YANG file or a directory with YANG files which `gnnmic` will use to generate auto-completion for YANG-defined data (paths, models).
 
 Multiple `--file` flags can be supplied.
 
 #### dir
-
-Direcotries where to search for yang modules included and imported in modules under `--file`
+A path to a directory which `gnmic` would recursively traverse in search for the additional YANG files which may be required by YANG files specified with `--file` to build the YANG tree.
 
 Multiple `--dir` flags can be supplied.
 
 #### exclude
-
-The `--exclude` flag is used to specify the yang modules to be excluded from path generation
+The `--exclude` flag specifies the YANG modules to be excluded from the path generation. This is used in the situations when YANG modules names are clashed.
```

Comment:
Let's specify `YANG modules __names__ to be excluded` since it's not a file.
At the end I think `YANG modules names clash` is enough

Reply:
done